Formula/rename.rb

```diff
@@ -3,6 +3,9 @@ class Rename < Formula
   homepage "http://plasmasturm.org/code/rename"
   url "https://github.com/ap/rename/archive/v1.600.tar.gz"
   sha256 "538fa908c9c2c4e7a08899edb6ddb47f7cbeb9b1a1d04e003d3c19b56fcc7f88"
+
```

Comment:
Actually we do not want an extra blank line prior to `revision`, but I will drop that blank line upon merging.